### DISCUSSION TITLE
The session ssh2 disconnect itself after inactivity. : standbyime

### DISCUSSION THREAD

**Discussion by AnkushVats-Ahold-GET** [mRemoteNG/mRemoteNG#2638] [General]
My ssh2 session disconnect automatically after 2 minutes itself. Can we set this ideal time or standby time in mremoteng, if yes, please share the steps.

**Comment by Kvarkas** [comment]
hello, what cannot be controlled from mR side, that seems to be a policy on remote server. You may try another server to verify
